examples/widgetbook_example/widgetbook/widgetbook.dart

```diff
@@ -5,6 +5,7 @@ import 'package:widgetbook/widgetbook.dart';
 import 'package:widgetbook_example/models/meal.dart';
 import 'package:widgetbook_example/themes/dark_theme.dart';
 import 'package:widgetbook_example/themes/light_theme.dart';
+import 'package:widgetbook_example/widgets/attributes/multiline_knob.dart';
```

Comment:
This looks like your code is not up to date regarding the `base` branch. `git merge origin/widgetbook-3` should solve this.

Reply:
Unfortunately merge is not resolving this, it seems that this change is coming from the `widgetbook-3` branch

<img width="671" alt="image" src="https://user-images.githubusercontent.com/50345358/209636032-d9e2903b-1176-4622-b34a-67890d9139fc.png">

Reply:
Ok, we can merge it with this PR.

Reply:
Resoved